packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart

```diff
@@ -630,6 +630,7 @@ Map<String, Object> _jsonForMapConfiguration(MapConfiguration config) {
     if (config.trafficEnabled != null) 'trafficEnabled': config.trafficEnabled!,
     if (config.buildingsEnabled != null)
       'buildingsEnabled': config.buildingsEnabled!,
+    if (config.cloudMapId != null) 'cloudMapId': config.cloudMapId!,
```

Comment:
Same here.

Reply:
Added dart unit test to check that cloudMapId is passed through the PlatformView creation message.